presto-orc/src/main/java/com/facebook/presto/orc/TupleDomainOrcPredicate.java

```diff
@@ -124,6 +126,9 @@ else if (type.getJavaType() == double.class && columnStatistics.getDoubleStatist
         else if (type.getJavaType() == Slice.class && columnStatistics.getStringStatistics() != null) {
             return createDomain(type, hasNullValue, columnStatistics.getStringStatistics());
         }
+        else if (FLOAT.equals(type) && columnStatistics.getDoubleStatistics() != null) {
```

Comment:
Are there plans to support float type natively in Orc (introduce FloatStatistics etc.)? IIRC ORC format supports `float` in addition to `double`.